tests/test_export.py

```diff
@@ -254,57 +247,3 @@ def test_export_with_category_path_failure(self):
                 exporter.export()
 
 
-class SanitizeHtmlTests(unittest.TestCase):
```

Comment:
Hehe, ok -- well do you want me to merge this PR and make a new one, or would you rather keep this PR open until you write the tests?

BTW - do you see a button at the bottom entitled "squash and merge", once everything is reviewed and CI approved, please feel free to merge yourself if you do see that button.